CHANGELOG.md

```diff
@@ -0,0 +1,31 @@
+# Changelog
+All notable changes to this project will be documented in this file. If you make a notable change to the project, please add a line describing the change to the "unreleased" section. The maintainers will make an effort to keep the [Github Releases](https://github.com/NREL/OpenOA/releases) page up to date with this changelog. The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
+
+## [Unreleased]
+- Energy Yield Analysis (EYA) to Operational Assessment(OA) Gap Analysis method
+- Uncertainty quantification
+- Implemented open source Engie example data, and complete update of example notebooks
```

Comment:
Is the unreleased category intended to describe the develop branch or just master? These additions are or will soon be part of develop, but not on master yet. Unless if this is referring to the old API method of accessing Engie data.

Reply:
develop branch